src/components/partials/profile/plugins/GovernancePlugin.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <Accordion icon="file-tray-full-outline.svg">
+        <template #title>GOVERNANCE</template>
+        <template #body>
+            <div class="mb-3" v-for="(governance, i) in governance" :key="i">
+                <div v-if="i < limitBy">
+                    <p
+                        class="
+                            fw-light
+                            text-muted
+                            border border-1
+                            rounded-2
+                            p-2
+                            w-50
+                        "
+                    >
+                        Block #{{ governance.block }}
+                    </p>
+                    <ProfileActivity
+                        v-if="governance.type === 'PRO_VOTE'"
+                        icon="happy-outline.svg"
+                        message="voted yay on Referenda #24"
+                    />
+
+                    <ProfileActivity
+                        v-if="governance.type === 'TREASURY'"
+                        icon="wallet.svg"
+                        message="tipped on treasury tip with reason 'reason'"
```

Comment:
please add an util (extra util script) method "formatActivityText" with one parameter GovernanceObject and returns a string (the text to show) ... can be hardcoded in the method for now.